From 6e09c1d71d23d4c17f035577b318b873960db1d8 Mon Sep 17 00:00:00 2001
From: Richard Biener <rguenther@suse.de>
Date: Thu, 22 Dec 2022 15:51:46 +0100
Subject: [PATCH 311/547] bootstrap/106482 - document minimal GCC version

There's no explicit mention of what GCC compiler supports C++11
and the cross compiler build requirement mentions GCC 4.8 but not
GCC 4.8.3 which is the earliest known version to not run into
C++11 implementation bugs.  The following adds explicit wording.

	PR bootstrap/106482
	* doc/install.texi (ISO C++11 Compiler): Document GCC version
	known to work.

(cherry picked from commit b97c33fbd28e74a206c81c96a9b0b9fa3c8545d6)
---
 gcc/doc/install.texi | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/gcc/doc/install.texi b/gcc/doc/install.texi
index cd9607b2759..19d073256a9 100644
--- a/gcc/doc/install.texi
+++ b/gcc/doc/install.texi
@@ -227,7 +227,9 @@ described below.
 @heading Tools/packages necessary for building GCC
 @table @asis
 @item ISO C++11 compiler
-Necessary to bootstrap GCC.
+Necessary to bootstrap GCC.  GCC 4.8.3 or newer has sufficient
+support for used C++11 features, with earlier GCC versions you
+might run into implementation bugs.
 
 Versions of GCC prior to 11 also allow bootstrapping with an ISO C++98
 compiler, versions of GCC prior to 4.8 also allow bootstrapping with a
@@ -236,7 +238,7 @@ bootstrapping with a traditional (K&R) C compiler.
 
 To build all languages in a cross-compiler or other configuration where
 3-stage bootstrap is not performed, you need to start with an existing
-GCC binary (version 4.8 or later) because source code for language
+GCC binary (version 4.8.3 or later) because source code for language
 frontends other than C might use GCC extensions.
 
 @item C standard library and headers
-- 
2.25.1

